pkg/identityserver/mock/gateway_registry.go

```diff
@@ -28,6 +28,24 @@ import (
 
 var errNoGatewayRights = errors.DefinePermissionDenied("no_gateway_rights", "no gateway rights")
 
+func DefaultGateway(ids ttnpb.GatewayIdentifiers) *ttnpb.Gateway {
```

Comment:
You may options (`WithLocationPublic`, `WithUpdateLocationFromStatus`) to this call in order to avoid having to customize the gateway after the call.